packages/preview/use-tabler-icons/0.2.0/README.md

```diff
@@ -0,0 +1,57 @@
+> [!NOTE]
+>
+> This project is greatly inspired by and mainly edited based on [typst-fontawesome](https://github.com/duskmoon314/typst-fontawesome).
+
+<h1 align="center">
+  <img alt="Typst" src="assets/splash.svg">
+</h1>
```

Comment:
Hi @elegaanz, could you please take a look at this and provide some help?

*I thought I had left this comment when I opened the PR, but I just realized I didn't submit the code review - only to leave it "pending" and only visible to myself 😂*

Reply:
HTML is supported in Universe (well, a safe subset of it), but `<h1>` tags are removed, because there should only be one per page for accessibility and SEO reasons. Try to wrap the image in a `<h2>` or something else, it should fix the issue. You can submit a PR that just changes that without making a new version.

Reply:
> Try to wrap the image in a `<h2>` or something else, it should fix the issue. You can submit a PR that just changes that without making a new version.

I'll try to fix it in the next release. Thank you for the advice!